### PR TITLE
September changes

### PR DESCRIPTION
**Pull request by priyankaSahutekdi**

### CODE REVIEW

File: automation/src/main/java/com/ucm/config/BaseClass.java [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    	JavascriptExecutor js3 = (JavascriptExecutor) driver;
+		js3.executeScript("window.scrollBy(0,90000)");
+	}
+    
+    public void scrollDown4() {
+    	JavascriptExecutor js4 = (JavascriptExecutor) driver;
+		js4.executeScript("window.scrollBy(0,200000)");
+	}
+    
+    public void scrollUp1() {
```
Comment (Kaustubharas) [inline automation/src/main/java/com/ucm/config/BaseClass.java:88]: @priyankaSahutekdi - All the three methods (scrollUp1, scrollUp2. scrollUp3) looks exactly same. Please keep only one and remove the duplicate ones.

File: automation/src/main/java/com/ucm/config/BaseClass.java [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+	
+	public void selectDropdownValue(WebElement name1, WebElement name2) {
+        WebDriverWait wait = new WebDriverWait(driver,20);
+		
+		name1.click();
+		wait.until(ExpectedConditions.visibilityOf(name2));
+		name2.click();
+	}
+	
+	public void scrollDown1() {
```
Comment (Kaustubharas) [inline automation/src/main/java/com/ucm/config/BaseClass.java:83]: @priyankaSahutekdi - Please keep only one scrollDown method and remove the duplicate methods.

File: automation/src/main/java/com/ucm/config/BaseClass.java [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+			driver = new ChromeDriver(options);
+			*/
+			driver = new ChromeDriver();
+			properties = new Properties();
+			FileInputStream fis = new FileInputStream(
+					System.getProperty("user.dir") + "/src/main/java/com/ucm/config/properties.properties");
+			properties.load(fis);
+	
+
+			// ExcelUtils_Old.setExcelFile(Constant.TESTDATAPATH, "LoginDetails");
```
Comment (Kaustubharas) [inline automation/src/main/java/com/ucm/config/BaseClass.java]: @priyankaSahutekdi - remove commented code from the scripts. Only documentation comments are allowed.

File: automation/src/main/java/com/ucm/pageobjects/AdminUcmFormCreationPage.java [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+	private WebDriver driver;
+	static Logger log = Logger.getLogger(AdminUcmFormCreationPage.class);
+
+	public AdminUcmFormCreationPage(WebDriver driver) {
+		this.driver = driver;
+		PageFactory.initElements(driver, this);
+	}
+	/*
+	 * Locators for
```
Comment (Kaustubharas) [inline automation/src/main/java/com/ucm/pageobjects/AdminUcmFormCreationPage.java]: @priyankaSahutekdi - Please complete all the incomplete Documentation comments.

File: automation/src/main/java/com/ucm/config/BaseClass.java (hunk 1 above)
Comment (priyankaSahutekdi) [reply automation/src/main/java/com/ucm/config/BaseClass.java:88]: done

File: automation/src/main/java/com/ucm/config/BaseClass.java (hunk 2 above)
Comment (priyankaSahutekdi) [reply automation/src/main/java/com/ucm/config/BaseClass.java:83]: done

File: automation/src/main/java/com/ucm/config/BaseClass.java (hunk 3 above)
Comment (priyankaSahutekdi) [reply automation/src/main/java/com/ucm/config/BaseClass.java]: done

File: automation/src/main/java/com/ucm/pageobjects/AdminUcmFormCreationPage.java (hunk 4 above)
Comment (priyankaSahutekdi) [reply automation/src/main/java/com/ucm/pageobjects/AdminUcmFormCreationPage.java]: done

**Review by Kaustubharas — APPROVED** [review]

**Comment by Kaustubharas** [comment]
reviewed

**Pull request merged**